### DISCUSSION TITLE
Hide scores during gameplay ("TAB" function on Stable)

### DISCUSSION THREAD

**Discussion by Bryceed** [ppy/osu#23235] [Ideas]
For thoses who prefers a more cleaner interface to focus, but without losing all informations on the screen, the implementation of that [ TAB ] function, to hide ranking scores during the gameplay like in stable, will be something really appreciated for the community.

Current usage:
![osu_2023-04-19_12-49-57](https://user-images.githubusercontent.com/42657376/233131319-7d1ab10c-7fac-4abd-9362-0e8473b735f7.jpg)

Expected:
![osu_2023-04-19_12-50-53](https://user-images.githubusercontent.com/42657376/233131386-edbd4eea-68b7-42b5-9004-aad3e0ca71a3.jpg)

**Comment by Bryceed** [comment]
...also, it's possible to have something similar today, but just on unranked maps or when, sometimes, the ranking aren't loaded.

**Reply by Theighlin** [reply]
The leaderboard reflects the one selected in song select (global/country/local) and you can disable it in the settings for singleplayer for now.
